gen/convert.go

```diff
@@ -49,6 +49,10 @@ func toGoType(ipaType string) string {
 		return "string"
 	case "Decimal":
 		return "float64"
+	case "Certificate":
+		return "interface{}"
```

Comment:
```suggestion
		return "string"
```

Certificates had type `bytes` in the old schema ([diff](https://gist.github.com/tehwalris/3b74d9e9d484e2ae62039febca29277b#file-schema-diff-L523)), so the generated fields had type `string`. We should avoid changing that if possible. Is there a reason why you went with `interface{}` here?

Reply:
Yes, there is a reason. Certificates are actually modeled as maps with a `__base64__` subfield, so casting them as strings failed.